src/EventHub/EventHub.Test/ScenarioTests/NetworkRuleSetTests.ps1

```diff
@@ -0,0 +1,115 @@
+	$result =  Add-AzEventHubVirtualNetworkRule -ResourceGroup $resourceGroupName -Name $namespaceName -SubnetId "/subscriptions/854d368f-1828-428f-8f3c-f2affa9b2f7d/resourcegroups/v-ajnavtest/providers/Microsoft.Network/virtualNetworks/sbehvnettest1/subnets/sbdefault"
+	$result =  Add-AzEventHubVirtualNetworkRule -ResourceGroup $resourceGroupName -Name $namespaceName -SubnetId "/subscriptions/854d368f-1828-428f-8f3c-f2affa9b2f7d/resourcegroups/v-ajnavtest/providers/Microsoft.Network/virtualNetworks/sbehvnettest1/subnets/sbdefault01"
+
+	Write-Debug "Get NetwrokRuleSet"
+	$getResult1 = Get-AzEventHubNetworkRuleSet -ResourceGroup $resourceGroupName -Name $namespaceName
+	
+	Assert-AreEqual $getResult1.VirtualNetworkRules.Count 3 "VirtualNetworkRules count did not matched"
+	Assert-AreEqual $getResult1.IpRules.Count 3 "IPRules count did not matched"
+
+	Write-Debug "Remove a new IPRule to the default NetwrokRuleSet"
+    $result =  Remove-AzEventHubIPRule -ResourceGroup $resourceGroupName -Name $namespaceName -IpMask "3.3.3.3"	
+
+	$getResult = Get-AzEventHubNetworkRuleSet -ResourceGroup $resourceGroupName -Name $namespaceName
+
+	Assert-AreEqual $getResult.IpRules.Count 2 "IPRules count did not matched after deleting one IPRule"
+	Assert-AreEqual $getResult.VirtualNetworkRules.Count 3 "VirtualNetworkRules count did not matched"
+
+	# Set-AzEventHubNetworkRuleSet with InputObject
+	$setResult =  Set-AzEventHubNetworkRuleSet -ResourceGroup $resourceGroupName -Name $namespaceName2 -InputObject $getResult1
+	Assert-AreEqual $setResult.VirtualNetworkRules.Count 3 "Set -VirtualNetworkRules count did not matched"
+	Assert-AreEqual $setResult.IpRules.Count 3 "Set - IPRules count did not matched"
+
+	# Set-AzEventHubNetworkRuleSet with Resource ID
+	$setResult1 =  Set-AzEventHubNetworkRuleSet -ResourceGroup $resourceGroupName -Name $namespaceName2 -ResourceId $getResult.Id
+	Assert-AreEqual $setResult1.IpRules.Count 2 "Set1 - IPRules count did not matched after deleting one IPRule"
+	Assert-AreEqual $setResult1.VirtualNetworkRules.Count 3 "Set1 - VirtualNetworkRules count did not matched"
+
+	Write-Debug "Add a new VirtualNetworkRule to the default NetwrokRuleSet"
+    $result =  Remove-AzEventHubVirtualNetworkRule -ResourceGroup $resourceGroupName -Name $namespaceName -SubnetId "/subscriptions/854d368f-1828-428f-8f3c-f2affa9b2f7d/resourcegroups/v-ajnavtest/providers/Microsoft.Network/virtualNetworks/sbehvnettest1/subnets/default"
+	
+	Write-Debug "Delete NetwrokRuleSet"
+    $result =  Remove-AzEventHubNetworkRuleSet -ResourceGroup $resourceGroupName -Name $namespaceName   
+
+    Write-Debug " Delete namespaces"    
+    Remove-AzEventHubNamespace -ResourceGroup $resourceGroupName -Name $namespaceName
+
+	Write-Debug " Delete resourcegroup"
+	#Remove-AzResourceGroup -Name $resourceGroupName -Force
```

Comment:
@v-Ajnava any reason that this line is commented out? Tests usually have proper deconstruction to ensure that resources that were created during the tests are also destroyed